Form-Controls/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  width: 100%;
+  font-size: 20px;
+  color: black;
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: 0;
+  background: rgb(245, 206, 218);
+  padding: 0;
+}
+
+
+h1{
+  
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+input{
+  width: 400px;
+  height: 15px;
+  font-size: 20px;
+  padding: 10px;
+  border: 0px solid white;
+  border-radius: 8px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+}
+select{
+  width: 400px;
+  height: 40px;
+  font-size: 20px;
+  padding: 10px;
+  border: 0px solid white;
+  border-radius: 2px;
+  margin-top: 5px;
+  margin-bottom: 20px;
+  margin-left: 85px;
+}
+
+
+form{
+  width: 45%;
+  margin-left:auto;
+  margin-right: auto;
+  border:1px solid white;
+ border-radius: 3px;
+ padding: 20px;
+ 
+}
+
+.button{
+  height: 40px;
+  width: 150px;
+  font-size: 20px;
+  margin-bottom: 10px;
+  padding: 2px;
+  border: 0px solid #222222;
+  border-radius: 3px;
+  background: lightslategray; 
+  color: black;
+  cursor: pointer;
+  margin-left: 30%;
```

Comment:
Instead of using `margin-left: 30%` for alignment, center the button using flexbox inside `#btn-align`